test/e2e-tests.sh

```diff
@@ -121,6 +121,23 @@ go_test_e2e -timeout=2m ./test/e2e/initcontainers ${TEST_OPTIONS} || failed=1
 toggle_feature kubernetes.podspec-init-containers Disabled
 toggle_feature kubernetes.podspec-volumes-emptydir Disabled
 
+# RUN PVC tests with default storage class if available.
+if (( USE_DEFAULT_STORAGE )) ; then
+  kubectl apply -f "${E2E_YAML_DIR}/test/config/pvc/pvc.yaml" -n serving-tests
+fi
+
+toggle_feature kubernetes.podspec-persistent-volume-claim Enabled
+toggle_feature kubernetes.podspec-persistent-volume-write Enabled
+toggle_feature kubernetes.podspec-securitycontext Enabled
```

Comment:
Because my services needs to do: 
https://github.com/knative/serving/blob/bdf92a361aee28ddced2ae15b70015f11e54dd20/test/e2e/pvc/pvc_test.go#L62-L63

Reason is that `ko` uses 62532 and GCP runs tests with uid 1000+.  This leads to permission access issues.

Reply:
I'm confused - the Knative Service is deployed on a differerent cluster than were the tests run (prow).

Reply:
That cluster then creates a problem since the uid does not match ko. You can check above my changes trying to fix this to run on the CI clusters.

Reply:
What user should that be?